axelar-chains-config/src/utils/sourcify.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const axios = require('axios');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+async function findValue(obj, contractName, result = []) {
+    if (obj && typeof obj === 'object') {
+        for (const key in obj) {
+            if (key.includes(`/${contractName}.sol`) || key === contractName) {
+                result.push(obj[key].metadata ? obj[key].metadata : obj[key].content);
+            }
+
+            await findValue(obj[key], contractName, result);
+        }
+    }
+}
+
+async function readAndParseFile(filePath, targetKey) {
+    try {
+        const jsonData = await fs.promises.readFile(filePath, { encoding: 'utf8' });
+        const parsedData = JSON.parse(jsonData);
+        const result = [];
+        await findValue(parsedData, targetKey, result);
+        return result;
+    } catch (error) {
+        console.error(`Error reading or parsing file ${filePath}: ${error.message}`);
```

Comment:
```suggestion
        printError(`Error reading or parsing file ${filePath}`, error.message);
```

Reply:
Using printError or printInfo in axelar-chains-config introduces circular dependency problem, should I declare it again in axelar-chains-config?
Also I think it is better to just use the normal console as done in other place like [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/dc8e9d6acbd6e26938006f5d273ca178fc14fa50/axelar-chains-config/src/utils/verifyContract.js#L35) .